lms/static/js/spec/verify_student/make_payment_step_view_ab_testing_spec.js

```diff
@@ -0,0 +1,221 @@
+define([
+        'jquery',
+        'underscore',
+        'backbone',
+        'common/js/spec_helpers/ajax_helpers',
+        'common/js/spec_helpers/template_helpers',
+        'js/verify_student/views/make_payment_step_view'
+    ],
+    function( $, _, Backbone, AjaxHelpers, TemplateHelpers, MakePaymentStepView ) {
+        'use strict';
+
+        describe( 'edx.verify_student.MakePaymentStepView', function() {
+
+            var PAYMENT_PARAMS = {
+                orderId: "test-order",
+                signature: "abcd1234"
+            };
+
+            var STEP_DATA = {
+                minPrice: "12",
+                currency: "usd",
+                processors: ["test-payment-processor"],
+                courseKey: "edx/test/test",
+                courseModeSlug: 'verified',
+                isABTesting: true
+            };
+
+            var SERVER_ERROR_MSG = "An error occurred!";
+
+            var createView = function( stepDataOverrides ) {
+                var view = new MakePaymentStepView({
+                    el: $( '#current-step-container' ),
+                    stepData: _.extend( _.clone( STEP_DATA ), stepDataOverrides ),
+                    errorModel: new ( Backbone.Model.extend({}) )()
+                }).render();
+
+                // Stub the payment form submission
+                spyOn( view, 'submitForm' ).andCallFake( function() {} );
+                return view;
+            };
+
+            var expectPriceSelected = function( price ) {
+                var sel = $( 'input[name="contribution"]' );
+
+                // check that contribution value is same as price given
+                expect( sel.length ).toEqual(1);
+                expect( sel.val() ).toEqual(price);
+            };
+
+            var expectPaymentButtonEnabled = function( isEnabled ) {
+                var el = $( '.payment-button'),
+                    appearsDisabled = el.hasClass( 'is-disabled' ),
+                    isDisabled = el.prop( 'disabled' );
+
+                expect( !appearsDisabled ).toEqual( isEnabled );
```

Comment:
It would be clearer what the test is for if you used the following Jasmine syntax
```expect( appearsDisabled ).not.toEqual( isEnabled );```